web_client/src/components/ExperimentsView.vue

```diff
@@ -158,6 +158,9 @@ export default {
           const newExperiment = await djangoRest.createExperiment(
```

Comment:
Could we just have `createExperiment` throw its own error if something goes wrong? It could throw a custom error type that contains relevant info.

Reply:
Changed in [ffed89e](https://github.com/OpenImaging/miqa/pull/563/commits/ffed89ed8303d5412e440f273887e9e308e2d928)